common/AdminUtils.js

```diff
@@ -37,8 +37,14 @@ function getAdminRequestId(identifierUtf8) {
   return parseInt(identifierUtf8.slice(adminPrefix.length), 10);
 }
 
+// Vote 1 for Yes, anything else for No.
+const translateAdminVote = voteValue => {
+  return voteValue === "1" ? "YES" : "NO";
```

Comment:
nit: I think this is _slightly_ inaccurate. Technically, the Governor interprets any nonzero value as YES.

See https://github.com/UMAprotocol/protocol/blob/b04c2e2fa1f94709e9952a8728c3e7b7ef91244f/core/contracts/oracle/implementation/Governor.sol#L153.

However, if someone manually votes `3` (outside of the dapp) and everyone else votes `1`, we don't want to classify those both as YES in the dapp leading the voter to think they voted correctly and deserve rewards. All that is to say, maybe we should do something like this:
```
switch(voteValue) {
  case "1":
    return "YES";
  case "0":
    return "NO";
  default:
    return "INVALID ADMIN VOTE";
}
```

Thoughts?

Reply:
So let me confirm my understanding:
- any non zero answer is a YES, but that means there many different ways to say YES
- only one YES value is the correct vote (I.e. “1” is YES and is the correct vote, while “3” is also YES but is not the correct vote)

If this is correct then I agree with your 3 cases

Reply:
That’s correct.

Reply:
Implemented